x-pack/test/apm_api_integration/tests/mobile/mobile_main_statistics_by_field.spec.ts

```diff
@@ -102,14 +219,14 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         const fieldValues = response.mainStatistics.map((item) => item.name);
 
-        expect(fieldValues).to.be.eql(['10']);
+        expect(fieldValues).to.be.eql(['10', '11']);
```

Comment:
Perhaps for consistency
```suggestion
        expect(fieldValues).to.be.eql(osVersions);
```